contents/blog/posthog-vs-ga4.mdx

```diff
@@ -34,7 +32,7 @@ In this article we’ll explore these differences in more detail, and take a dee
 
 ### 2. PostHog is built for engineers
 
-Anyone can use PostHog – just [ask our marketing team](/blog/posthog-marketing) – but PostHog is designed to meet the needs of engineers. Our goal is to help developers [be better at product](/blog/helping-engineers-to-product).
+Anyone can use PostHog – just [ask our marketing team](/blog/posthog-marketing) – but PostHog is designed to meet the needs of engineers. Our goal is to help developers [be better at product](/blog/helping-engineers-to-product). We offer both light and dark mode UIs, and power user features like custom SQL insights.
```

Comment:
Same-ish point as above

> Autocapture and dark mode seems like something non-technical people would appreciate, rather than engineers.
> Engineers would care more about SDKs, simple installation for multiple integrated tools, API, HogQL, EU hosting.